### DISCUSSION TITLE
unable to verify the first certificate

### DISCUSSION THREAD

**Discussion by Swiftlyfisher** [prisma/prisma#19852] [Q&A]
![image](https://github.com/prisma/prisma/assets/57713868/1305b6ca-abaf-46b3-b744-4acca2725476)

**Comment by Swiftlyfisher** [comment]
Does vpn have any effect on it? But I still get the same error when I turn off vpn

**Reply by ludralph** [reply]
Hi @Swiftlyfisher 👋 

Using a VPN (Virtual Private Network) can sometimes impact the certificate verification process, but simply having a VPN enabled shouldn't cause the specific error message you mentioned ("unable to verify the first certificate"). The error typically relates to the SSL/TLS certificate validation on the server side, rather than being directly related to your network configuration. if you turn off the VPN and still encounter the same error, it suggests that the issue lies elsewhere and is unrelated to the VPN connection.

To resolve this issue, you can try the following steps:

1. Check your system's date, time, and timezone settings and ensure they are accurate.

2. Try accessing the website or server using a different browser or device to see if the issue persists.


If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by janpio** [comment]
Can you download the file via the URL that is output manually in your browser? 
What if you do `curl https://....` on the command line?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
